### PR TITLE
METAL-1847 support compatible key with stable helm for admin secret

### PR DESCRIPTION
**Pull request by hyunysmile**
admin secret is compatible with secret generated by stable helm chart for database creation

### CODE REVIEW

File: pkg/utils/database/mysql.go [hunk 1]
```diff
@@ -195,3 +196,33 @@ func (m Mysql) GetCredentials() Credentials {
...
+	_, ok := data["user"]
+	if ok {
+		cred.Username = string(data["user"])
+	} else {
+		// default admin username is "root"
+		cred.Username = "root"
+	}
+
+	// if "password" key is defined in data, take value as password
+	if ok {
```
Comment (dabde) [inline pkg/utils/database/mysql.go:214]: the check if password exist is missing, it's take the check from the username.

File: pkg/utils/database/mysql.go [hunk 2]
```diff
@@ -195,3 +196,33 @@ func (m Mysql) GetCredentials() Credentials {
 		Password: m.Password,
 	}
 }
+
+// ParseAdminCredentials parse admin username and password of mysql database from secret data
+// If "user" key is not defined, take "root" as admin user by default
+func (m Mysql) ParseAdminCredentials(data map[string][]byte) (AdminCredentials, error) {
```
Comment (dabde) [inline pkg/utils/database/mysql.go:202]: @hyunysmile what do you think to add the keys for password/username to the instance as properties, so it's dynamic in the future and we only fallback to "user" and "password", if not set.

(minor)

File: pkg/utils/database/mysql.go (hunk 1 above)
Comment (hyunysmile) [reply pkg/utils/database/mysql.go:215]: oops thanks for catching it

File: pkg/utils/database/mysql.go (hunk 2 above)
Comment (hyunysmile) [reply pkg/utils/database/mysql.go:202]: we can think about it when there are more cases that we need to handle.

**Pull request merged**